creator-node/package.json

```diff
@@ -22,7 +22,7 @@
   "dependencies": {
     "@audius/sdk": "0.0.15",
     "@bull-board/express": "4.0.1",
-    "@solana/web3.js": "1.31.0",
+    "@solana/web3.js": "1.53.0",
```

Comment:
scary but we will test

Reply:
def agreed. if its too scary there is a way to force that version in native to reduce footprint

Reply:
i can also double check with older version too